src/main/java/org/apache/datasketches/memory/WritableMemory.java

```diff
@@ -83,54 +83,50 @@ static WritableMemory writableWrap(
    * required by the implementation.
    */
   static WritableMemory writableMap(File file) throws IOException {
-    return writableMap(file, 0, file.length(), ByteOrder.nativeOrder());
+    return writableMap(Arena.ofConfined(), file, 0, file.length(), ByteOrder.nativeOrder());
   }
 
   /**
-   * Maps the specified portion of the given file into Memory for write operations.
-   * @param file the given file to map. It must be non-null and writable.
+   * Maps the specified portion of the given file into Memory for write operations with a ResourceScope.
```

Comment:
I think you meant _Arena_, but that isn't correct either.  The write operations of WritableMemory assume that the MemorySegment exists and has already been allocated using an Arena.